migration/sql-files/022-add-deprovisioned-to-identity.sql

```diff
@@ -0,0 +1 @@
+ALTER TABLE identities ADD COLUMN deprovisioned BOOLEAN;
```

Comment:
so postgres should default it to `false`, right?

Reply:
I would expect NULL. But that's OK.

Reply:
It will be false by default in account.Identity though.

Reply:
Oh yeah, NULL https://www.postgresql.org/docs/9.1/static/datatype-boolean.html
Let's set it to false by default then. Will help us if we have to ever collect data from the db directly :)

Reply:
I don't think it's a problem to construct a query if we want to collect all false/null or true records :)
Just don't want to add unnecessary data to the table.
@sbryzak maybe there are some good practices for such cases?

Reply:
Do we want the NULL value to imply something like `we don't know if user was deprovisioned` ? If yes, let's keep it NULL. 
Or, if we plan to imply a NULL as `user has not been deprovisioned`, then let's make it `false`

Reply:
Btw, could you please add a migration test for this.

Reply:
Currently it's not a big deal to update our table and set false for existing identities. We would like to avoid it when we have too big identity table/DB. So, again I still don't see any issue with treating NULL or FALSE as - not deprovisioned.
Basically if deprovisioned == true then the user is deprovisioned. If not then not. :) Our code works well and handles it properly.
But I will add this "false" assignment to the migration step this time to make our table looks "better" because we can afford it at this point.

Reply:
Actually our identities are quite generic objects in our DB. They can represent remote identities (GitHub, JIRA, etc) or our internal identities (org, user, etc). Not for all of them "deprovisioned" makes sense. So, let's keep it nullable. I've added the migration tests though.

Reply:
Sorry I didn't realize we had altered IDENTITY table. Why aren't we adding this column to USER table? Because that's where we store cluster information.

Reply:
I was thinking about both options.. To store it in Users or in Identities. Identities seemed to be "closer" to the identity provider created by OSO... But I'm not sure I chose the right option :)

Reply:
If we are going to support multiple clusters per user then it makes sense to keep deprovisioned in the identity. Maybe we will have to move cluster to the identity too. Not sure what's best for now though.